src/docs/truffle/getting-started/writing-external-scripts.md

```diff
@@ -80,7 +80,7 @@ know how we can improve it!
      // config_[1] starts remaining parameters.
      if (config.help) {
        console.log(`Usage: truffle run hello [name]`);
-       done(null, [], []);	
+       done(null, [], []);
```

Comment:
My editor made this spacing change, just noticed it. Is this line a typo anyway though?

Reply:
This line looks like a typo to me, `done` isn't part of the method signature. @gnidan @CruzMolina

Reply:
@eggplantzzz Ok .. removing.